src/components/new-request/search.vue

```diff
@@ -242,47 +242,49 @@
 
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
-        <BusinessLookupFetch ref="MyBusinessLookup"/>
+        <BusinessLookupFetch ref="MyBusinessLookup" />
         <SocietiesInfo
-           v-if="isSocietyDisabled && showSocietiesInfo"
-           :type="'restore'"
+          v-if="isSocietyDisabled && showSocietiesInfo"
+          :type="'restore'"
           :showDialog="showSocietiesInfo"
         />
-        <CompanyType v-if="getSearchBusiness && isBcRestorable && isSupportedRestoration(getEntityTypeCd) && !isSocietyDisabled" />
-        <Jurisdiction
-          v-if="isSelectedXproAndRestorable && !isSocietyDisabled"
-          cols="12"
-          md="4"
-        />
-
-        <!-- federal sub-flow -->
-        <XproFederalBullets v-if="isFederal && getSearchBusiness" />
-
-        <template v-if="showRestoreNameInput && !isSocietyDisabled">
-          <v-col
-            cols="12"
-            :md="(showDesignation || isSelectedXproAndRestorable) ? '8' : '12'"
-          >
-            <NameInput
-              :is-mras-search="(isXproFlow && isMrasJurisdiction && !getHasNoCorpNum)"
-              @emit-corp-num-validity="corpNumValid = $event"
-            />
-          </v-col>
-          <Designation
-            v-if="showDesignation"
+        <template v-else>
```

Comment:
As above, I'm using an else here to NOT display the following when the dialog is shown.